templates/WhoIAm/index.html.twig

```diff
@@ -67,19 +66,19 @@
         </div>
     </div>
 
-    <div class="row">
-        <h3 class="offset-2 mr-4 font-italic">
+    <div class="col-4 d-flex flex-row offset-md-4 my-3">
+        <h3 class="mr-md-4 font-italic">
             Mes compétences
         </h3>
-        <img src="/build/path.png" alt="Selfie de Peter" style="width: 100px">
+        <img src="/build/path.png" alt="flèche descendante" style="width: 100px">
     </div>
 
     <div class="row">
-        <div class="col-6">
+        <div class="col-12 col-md-6 d-flex align-items-center flex-column">
             <img src="/build/whoiam/image1.png" alt="Peter et une coachée" class="w-75">
```

Comment:
col-12 ne sert à rien, il l'est par défault.

Reply:
il y en a plusieurs - je n'ai noté que celui-ci... faudra que tu rebalaies la page :)